lib/pubsub/iam.js

```diff
@@ -0,0 +1,142 @@
+/*!
+ * Copyright 2014 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*!
+ * @module pubsub/iam
+ */
+
+'use strict';
+
+/**
+ * @type {module:common/util}
+ * @private
+ */
+var util = require('../common/util');
+
+/*! Developer Documentation
+ *
+ * @param {module:pubsub} pubsub - PubSub Object
+ */
+/**
+ * An Iam object allows you to manage a Resource's idenity and access.
+ *
+ * @constructor
+ * @alias module:pubsub/iam
+ *
+ * @example
+ * var pubsub = gcloud.pubsub({
+ *   projectId: 'grape-spaceship-123'
+ * });
+ *
+ * var topic = pubsub.topic('my-topic');
+ * var iam = topic.iam;
+ */
+function Iam(pubsub, resource) {
```

Comment:
I'm smh at myself for saying this, but I think we want `IAM` here, like with DNS. It should only go lower case if the other letters aren't part of an acronym or new word.